### ISSUE TITLE
terminal chat shows up when I start voice session in panel chat

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#207367]
1. open a terminal
2. show the panel chat
3. click the mic icon in the panel chat
4. 🐛 a badly rendered terminal chat widget shows up in the terminal

**Comment by meganrogge** [comment]
![Image](https://github.com/microsoft/vscode/assets/29464607/38507f5d-9c82-44e2-b781-3263ff2a5950)

**Comment by meganrogge** [comment]
this was only happening on a branch where I had made a change to the voice actions to try to fix https://github.com/microsoft/vscode-copilot/issues/4498

**Comment by Tyriar** [comment]
Saw this in insiders today

**Comment by Tyriar** [comment]
Another report: https://github.com/microsoft/vscode/issues/207536#issue-2183092299

**Comment by meganrogge** [comment]
trying to find a repro

**Comment by meganrogge** [comment]
I see this but not the broken widget when I created a terminal and clicked on the chat panel's record button

```
voiceChatActions.ts:468 Uncaught ReferenceError: Cannot access 'session' before initialization
    at voiceChatActions.ts:468:3
    at async.ts:527:3
```

**Comment by meganrogge** [comment]
I wonder if we need to be focusing our chat input https://github.com/microsoft/vscode/blob/09882dea5dd5b151836219c1d197fac94a9c2c34/src/vs/workbench/contrib/chat/electron-sandbox/actions/voiceChatActions.ts#L639-L653

**Comment by Tyriar** [comment]
Can not repro for starting voice session specifically. For the general issue of this showing up I did a fix in #207751 which hopefully fixes it (no solid repro).
